winappsdk-workaround.targets

```diff
@@ -0,0 +1,27 @@
+<Project>
```

Comment:
If this file is specific to WinUI, can we move it under the WinUI directory instead of having it at the root of the repository?